getting-started-publishing/creating-descriptions.md

```diff
@@ -6,13 +6,60 @@ url: https://data.norge.no/getting-started/describing-data
 note: Vurder å slå sammen med "Publishing descriptions"
 ---
 
+Kanskje Terje kan skrive denne delen? 😇
+
 # Lage beskrivelser
 
+Det finnes to måter å lage databeskrivelser på: ved hjelp av vår [registreringsløsning](registrering.fellesdatakatalog.digdir.no) - da beskriver du hver datakilde maunelt, eller ved å lage en fil som er tilgjengelig på nett.
+
+## Registreringsløsning - manuell registreing av beskrivelser
+
+Dersom du ønsker å lage en og en beskrivelse manuelt, kan dette gjøres ved hjelp av vår [registreingsløsning](registrering.fellesdatakatalog.digdir.no).
+
+Når du har fått [tilgangene]() som kreves kan du logge inn på https://registrering.fellesdatakatalog.digdir.no. Deretter klikker du på type katalog du ønsker å registrere beskrivelser i, for eksempel datasettkatalog. Trykk deretter på "Legg til ny", og fyll ut skjemaet med relevant informasjon. Beskrivelsen din blir da lagret i katalogen. Katalogen blir automatisk høstet og endringer publisert hver hele time.
+
 ## Forarbeid: rammeverk for informasjonsforvaltning og Orden i eget hus
 
-## Registreringsløsning
+Alle kataloger og ressurser må ha en Unik global identifikator (URI). En Uniform Resource Identifier (URI) er en kompakt streng med tegn
+for å identifisere en abstrakt eller fysisk ressurs. Kort om URI: Kilde: http://www.faqs.org/rfcs/rfc2396.html.
+Andre mulige kilder: https://www.w3.org/TR/cooluris/#semweb, https://www.w3.org/2001/09/rdfprimer/section2.html#ref-uri
+https://joinup.ec.europa.eu/collection/semic-support-centre/document/10-rules-persistent-uris
+
+### Om validatoren
+
+Data.norge.no tilbyr en [validator](https://data.norge.no/validator) den du kan valdiere at filen din er skrevet korrekt. Her kan du laste opp RDF-grafen din, for å sjekke om syntax og format er i henhold til standardene.
+
+### Høsting
+
+Se mer om standardene her: Standard for beskrivelser av datasett,
+datatjenester og datakataloger: [DCAT-AP-NO](https://data.norge.no/specification/dcat-ap-no)
+
+Standard for beskrivelser av begrep: [SKOS-AP-NO-Begrep](https://data.norge.no/specification/skos-ap-no-begrep)
+
+Spesifikasjon for beskrivelser av informasjonsmodeller: [ModellDCAT-AP-NO](https://data.norge.no/specification/modelldcat-ap-no)
+
+Spesifikasjon for beskrivelser av tjenester og hendelser:
+[CPSV-AP-NO](https://informasjonsforvaltning.github.io/cpsv-ap-no/)
+
+#### RDF
+
+Syntakser som støttes:
+
+- turtle (https://www.w3.org/TR/turtle/)
+
+- rdf/xml (https://www.w3.org/TR/rdf-syntax-grammar/)
+
+- rdf/json (https://www.w3.org/TR/rdf-json/)
+
+- json-ld (https://www.w3.org/TR/json-ld11/), n-triples (https://www.w3.org/TR/n-triples/)
+
+- n3 (https://www.w3.org/TeamSubmission/n3/).
```

Comment:
```suggestion
- Notation3 / N3 (https://www.w3.org/TeamSubmission/n3/).
```

Reply:
Tror ikke vi støtter N3, Jena støtter det ikke: https://jena.apache.org/documentation/io/index.html

Reply:
De har et alternativ i Lang for N3, som vi bruker. Så nærmere på den nå, det blir tolket som Turtle. Så ja, tror vi fint bare kan fjerne alt ang. N3
```suggestion
```